.circleci/config.yml

```
@@ -0,0 +1,12 @@
+version: 2.1
+
+orbs:
+  dwave: dwave/orb-examples@2
+
+workflows:
+  version: 2.1
+  tests:
+    jobs:
+      - dwave/test-linux
+      - dwave/test-osx
+      - dwave/test-win
```

This just adds a new line at the end of the file

```suggestion
      - dwave/test-win

```